frontend/src/metabase/account/settings/components/AccountHeader.unit.spec.js

```diff
@@ -0,0 +1,47 @@
+import React from "react";
+import { fireEvent, render, screen } from "@testing-library/react";
+import AccountHeader from "./AccountHeader";
+
+const REGULAR_USER = {
+  id: 1,
+  first_name: "John",
+  last_name: "Doe",
+  email: "john@metabase.test",
+};
+
+const GOOGLE_USER = {
+  id: 2,
+  first_name: "John",
+  last_name: "Doe",
+  email: "john@metabase.test",
+  google_auth: true,
+};
+
+describe("AccountHeader", () => {
+  it("should show all tabs for a regular user", () => {
+    render(<AccountHeader user={REGULAR_USER} />);
+
+    expect(screen.getByText("Profile")).toBeInTheDocument();
+    expect(screen.queryByText("Password")).toBeInTheDocument();
+    expect(screen.getByText("Login History")).toBeInTheDocument();
+  });
+
+  it("should hide the password tab if it is disabled by a plugin", () => {
+    render(<AccountHeader user={GOOGLE_USER} />);
+
+    expect(screen.getByText("Profile")).toBeInTheDocument();
+    expect(screen.queryByText("Password")).not.toBeInTheDocument();
+    expect(screen.getByText("Login History")).toBeInTheDocument();
+  });
```

Comment:
Would be awesome to have a container test too just to make sure the plugin is used correctly. [Here's a unit-test example](https://github.com/metabase/metabase/blob/master/frontend/test/metabase/entities/collections/getCollectionIcon.unit.spec.js) that "patches" the plugins file, so the test is running in an EE-like environment

Reply:
Added a plugin test with beforeEach/afterEach block, as in the example. Thanks!